### PR TITLE
events: Use jemalloc to reduce RAM usage on Heroku

### LINKED ISSUE #573: Use jemalloc in the Heroku worker image
The improvements for bugbug were huge: https://github.com/mozilla/bugbug/issues/1421.

With enough tuning, we might be able to use Standard-1X (https://github.com/mozilla/bugbug/issues/1472).

### PR DESCRIPTION
**Pull request by La0**
Fixes #573 & clone of https://github.com/mozilla/bugbug/issues/1421

I'm pushing the PR to testing to test the memory reduction

### CODE REVIEW

File: events/docker/Dockerfile [hunk 1]
```diff
@@ -8,4 +8,7 @@ RUN cd /src/events && python setup.py install
 
 RUN /src/tools/docker/bootstrap-mercurial.sh
 
+# Setup jemalloc to manage Ram more efficiently on Heroku
+ENV LD_PRELOAD="libjemalloc.so.2"
+
```
Comment (marco-c) [inline events/docker/Dockerfile:13]: Could you try to set narenas to 2 and see if that makes a difference?
`export MALLOC_CONF="narenas:2"`

Comment (marco-c) [reply events/docker/Dockerfile:13]: Or even 1, since we have a single thread

Comment (La0) [reply events/docker/Dockerfile:14]: Done in a49ce55 & pushing to testing...

**Comment by La0** [comment]
Works on testing, with narenas=1, almost same result (~20% reduction in ram usage)

**Comment by La0** [comment]
Added recommended options to the env variable, memory usage at 1180Mb instead of 1.5Gb during clone.

**Pull request merged**